static/app/views/starfish/modules/databaseModule/panel/index.tsx

```diff
@@ -238,6 +248,27 @@ function QueryDetailBody({
             hideYAxisSplitLine
           />
         </FlexRowItem>
+        <FlexRowItem>
+          <SubHeader>{t('Top 5 Transactions by P75')}</SubHeader>
+          <Chart
+            statsPeriod="24h"
+            height={140}
+            data={p75TransactionSeries}
+            start=""
+            end=""
+            loading={isDataLoading}
```

Comment:
Did you forget to update this?
```suggestion
            loading={isP75GraphLoading}
```

Reply:
Yes I did! ty, nice catch!